neuralmonkey/train.py

```diff
@@ -183,6 +186,18 @@ def main() -> None:
         log(str(exc), color='red')
         exit(1)
 
+    if cfg.model.visualize_embeddings:
+
+        tb_projector = projector.ProjectorConfig()
+
+        for sequence in cfg.model.visualize_embeddings:
+            if not isinstance(sequence, EmbeddedFactorSequence):
```

Comment:
Tohle se bude muset časem změnit, protože do budoucna `EmbeddedFactorSequence` nebude jediná třída, která umí zacházet s embedingama, ale přestěhuje se do nějaké abstraktní třídy. S embeddingama ostatně zachází i rekurentní dekodér. Takže buď bych rovnou připravil toho rodiče (bude mít abstraktní property `vocabulary` typu `Vocabulary` a `embedding_matrix` typu `tf.Variable`) a nebo sem jenom prostě připiš TODO, že se to bude muset udělat pořádně.